SqlInstall.ps1

```diff
@@ -0,0 +1,556 @@
+﻿function Install-SqlServer ([string]$AppVolume, [string]$DataVolume, [string]$LogVolume, [string]$TempVolume, [string]$BackupVolume, [int]$Ctp)
```

Comment:
All of our commands start with Dba, so this would be `Install-DbaInstance`. The params are also laid out in a specific way. Take a look at `Get-DbaDatabase` to see what I'm talking about, down near [cmdletbinding()]

Reply:
I've changed the name of the function. 
Parameters are now in a seperate Param block. 
Further development will show if i've made the right choices :)

Reply:
awesome :D don't forget, if you commit to your same branch, it should show here and we can work through it as it happens.

Reply:
I've committed to this branch and so you should some progress.

Nowhere near a new pullrequest though ;)